gameswap-service/src/main/java/org/gameswap/application/GameswapService.java

```diff
@@ -115,25 +120,7 @@ private void addAuthFilter(Environment environment) {
 
 
     private void addHttpsForward(MutableServletContextHandler handler) {
-        handler.addFilter(new FilterHolder(new Filter() {
-
-            public void init(FilterConfig filterConfig) throws ServletException {
-            }
-
-
-            public void doFilter(ServletRequest request, ServletResponse response, FilterChain chain) throws IOException, ServletException {
-                StringBuffer uri = ((HttpServletRequest) request).getRequestURL();
-                if (uri.toString().startsWith("http://")) {
-                    String location = "https://" + uri.substring("http://".length());
-                    ((HttpServletResponse) response).sendRedirect(location);
-                } else {
-                    chain.doFilter(request, response);
-                }
-            }
-
-
-            public void destroy() {
-            }
-        }), "/*", EnumSet.allOf(DispatcherType.class));
+        handler.addFilter(new FilterHolder(new HttpsForwardingFilter()), "/*", EnumSet.allOf(DispatcherType.class));
```

Comment:
this could probably change to use the dropwizard helper like we are doing with the auth filter